azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/reducer/ErrorReducer.kt

```diff
@@ -12,9 +13,12 @@ internal interface ErrorReducer : Reducer<ErrorState>
 internal class ErrorReducerImpl : ErrorReducer {
     override fun reduce(state: ErrorState, action: Action): ErrorState {
         when (action) {
-            is ErrorAction.ChatStateErrorOccurred -> return state.copy(
-                chatCompositeErrorEvent = action.chatCompositeErrorEvent
-            )
+            is ErrorAction.ChatStateErrorOccurred -> {
```

Comment:
Logging an error is part of logic. We may extend it eventually and make it more complex.
We already have error handler class, why not to put logging there. Let's keep reducer a single responsibility to update state